src/App.scss

```
@@ -1 +1,34 @@
-// styles go here
+@import "styles/reset";
+@import "styles/variables";
+
+.app {
```

```suggestion
.App {
```

It's a BEM block, but as it is a React component use CamelCase